IFC4x3/Constants/l/LABTESTRESULT_2XsgKLRhL7Ngp7nQbm_mzA.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LABTESTRESULT_2XsgKLRhL7Ngp7nQbm_mzA" Name="LABTESTRESULT" UniqueId="a1daa515-6eb5-475e-acc7-c5a970fb0f4a">
+	<Documentation>Result from test on a rock/soil (geologic) or fluid specimen carried out in a laboratory</Documentation>
```

Comment:
<Documentation>Result from test on a rock/soil (geologic) or fluid specimen carried out in a laboratory.</Documentation>

Reply:
Fixed by @SergejMuhic